orienteer-core/src/main/java/org/orienteer/core/component/TabsPanel.java

```diff
@@ -47,7 +48,17 @@ public TabsPanel(String id, IModel<T> model, IModel<? extends List<T>> tabsModel
 
 			@Override
 			protected void populateItem(final ListItem<T> item) {
-				item.add(new AttributeAppender("class", "active")
+				AbstractLink link = new AjaxLink<T>("link", item.getModel()) {
+					@Override
+					public void onClick(AjaxRequestTarget target) {
+						TabsPanel.this.setModelObject(item.getModelObject());
+						onTabClick(target);
+						target.add(TabsPanel.this);
+					}
+				}.setBody(newTabNameModel(item.getModel()));
+				item.add(link);
+				
+				link.add(new AttributeAppender("class", " active")
```

Comment:
Is moving of classactive from `li` to `a` from Bootstrap?

Reply:
In coreui. Yep